README.md

```diff
@@ -53,15 +53,15 @@ The benefits:
 
 The downsides:
 
-- Heavier initial page load due to loading of framework, app code, and assets required for multiple pages [^1].
```

Comment:
I left it like this because it's the Jekyll footnote format. But I guess I should make it work here too. Could you help change it to use `<sup>[1](#ft-1)</sup>` instead? Using `*` will be kind of unscalable if we have more footnotes in future.